src/server/routes/api/document/hint/pubtator.js

```diff
@@ -12,18 +12,32 @@ function map(bioCDocument) {
 
   // See Table 1 https://www.ncbi.nlm.nih.gov/research/pubtator3/tutorial
   const PUBTATOR_ANNOTATION_TYPE = Object.freeze({
+    GENE: 'Gene',
     SPECIES: 'Species',
-    // could add more types here when scope expands
+    CHEMICAL: 'Chemical',
+    DISEASE: 'Disease',
+    CELL_LINE: 'CellLine',
+    // VARIANT: 'Variant',
   });
   const PUBTATOR_DATABASE = Object.freeze({
+    ncbi_gene: 'ncbi_gene',
     ncbi_taxonomy: 'ncbi_taxonomy',
-    // could add more databases here when scope expands
+    ncbi_mesh: 'ncbi_mesh',
+    cvcl: 'cvcl',
+    litvar: 'litvar',
   });
   const entityTypes = new Map([
+    [PUBTATOR_ANNOTATION_TYPE.GENE, HINT_TYPE.GGP],
     [PUBTATOR_ANNOTATION_TYPE.SPECIES, HINT_TYPE.ORGANISM],
+    [PUBTATOR_ANNOTATION_TYPE.CHEMICAL, HINT_TYPE.CHEMICAL],
+    [PUBTATOR_ANNOTATION_TYPE.DISEASE, HINT_TYPE.DISEASE],
+    [PUBTATOR_ANNOTATION_TYPE.CELL_LINE, HINT_TYPE.CELL_LINE],
   ]);
   const database2Xref = new Map([
+    [PUBTATOR_ANNOTATION_TYPE.GENE, COLLECTIONS.NCBI_GENE],
```

Comment:
I think this should be `PUBTATOR_DATABASE.ncbi_gene` rathan than `.GENE`

Reply:
Right, my bad